## Hi there ðŸ‘‹


**deppworld/deppworld** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.
Deepak Verma, PhD
depp@live.in; dverma2@jh.edu
116 W University Pkwy, Apt 501, Baltimore MD 21210 USA
+1 (443) 278-3518

*RESEARCH EXPERIENCE*
 
Postdoctoral Fellow 							                             1st February 2021 - present        
Johns Hopkins Medicine, Baltimore MD USA

â€¢	Leading high-throughput metagenomic and epigenomic data analysis focused on gut microbiome associations with breast cancer and obesity, with an emphasis on racial disparities in triple-negative breast cancer (TNBC).
â€¢	Designed and optimized experimental workflows for multi-omics studies, including NGS, RNA-Seq, scRNA-Seq, and methylation.
â€¢	Provided method development, data troubleshooting, and technical documentation to support cross-functional research teams and external collaborators.
â€¢	Delivered scientific presentations tailored to both Experts and lay audiences.
â€¢	Contributed to peer-reviewed publications and grant proposals through data interpretation and visualization.

Expertise: Metagenomics, Epigenomics, RNA-Seq, scRNA-Seq, 16S and Shotgun Sequencing, Experimental Design, Data Analysis, R, Python.

Research Associate 							                         1st July 2019 - 27th January 2021      
Laboratory Oncology, Dr. BRA-IRCH AIIMS, New Delhi, India 
â€¢	Performed high-throughput sequencing data analysis focusing on microRNA (miRNA) profiling in T-cell acute lymphoblastic leukemia (T-ALL).
â€¢	Designed and optimized experiments involving RNA-Seq (WTS), Exome Sequencing (WGS), Epigenomics, qPCR, and MLPA validation.
â€¢	Performed bioinformatics analysis, data integration, and interpretation using advanced computational tools and pipelines.
â€¢	Supported internal teams and external collaborators through technical documentation, troubleshooting, and project report preparation.
â€¢	Presented complex scientific findings in a clear and concise manner to both technical and non-technical professionals.

Expertise: NGS, Whole Genome Sequencing (WGS), Exome-Seq, Epigenomics, RNA-Seq, scRNA-Seq, qPCR, MLPA, Experimental Design, Data Analysis

Senior Research Fellow						                28th November 2018 - 30th June 2019   
Laboratory Oncology, Dr. BRA-IRCH AIIMS, New Delhi, India 
â€¢	Performed high-throughput data analysis for Indian BCR-ABL-like acute lymphoblastic leukemia (ALL) using copy number variation (CNV) and single nucleotide variant (SNV) profiling, and miRNA profiling in T-cell ALL.
â€¢	Led experimental design, pipeline development, and method optimization for multi-omics datasets including NGS, RNA-Seq, Exome-Seq, and epigenomic data.
â€¢	Conducted data preprocessing, variant calling, annotation, and visualization using bioinformatics tools and pipelines.
â€¢	Collaborated with cross-functional teams, ensuring accurate documentation, troubleshooting, and preparation of project reports and deliverables.
â€¢	Communicated results effectively through technical presentations, report submissions, and discussions with multidisciplinary teams and external collaborators.

Expertise: NGS, RNA-Seq, Epigenomics, Whole Genome/Exome Sequencing, CNV/SNV Analysis, qPCR, MLPA, Data Interpretation, Experimental Design

Senior Research Fellow 						                       1st August 2014 â€“ 28th April 2017
Laboratory Oncology, Dr. BRA-IRCH AIIMS, New Delhi, India 
â€¢	Contributing in high-throughput data analysis for Indian cytogenetically normal acute myeloid leukemia (AML) and miRNA profiling in T-cell acute lymphoblastic leukemia (T-ALL).
â€¢	Established a molecular lab for cancer diagnosis for testing including NGS.
â€¢	Led experimental design, method optimization, and implementation of various molecular biology techniques for cancer genomics studies.
â€¢	Managed end-to-end workflows including Sanger sequencing, PCR, Real-Time PCR, digital PCR (dPCR, ddPCR), and fragment analysis.
â€¢	Performed MLPA for key genes (e.g., IKZF in T-ALL), and supported analysis through flow cytometry, FISH, IHC, ICC, and immunoprecipitation (IP).
â€¢	Collaborated with multidisciplinary teams, ensured accurate documentation, troubleshooting, and report submission, and presented findings to both Experts and non-experts.
Expertise: Sanger Sequencing, PCR, Real-Time PCR, dPCR/ddPCR, MLPA, Flow Cytometry, FISH, IHC, ICC, IP, Fragment Analysis, Experimental Design, Data Analysis

EDUCATION
 
PhD in Field of Hemato-oncology & Bioinformatics	                                   		           12th April 2020
All India Institute of Medical Sciences					                               	          New Delhi, India
Molecular Biology of early immature T cell acute lymphoblastic leukemia.
PMID: 38328782; PMID: 35429905

M.S. (Bioinformatics)							                                 5th January 2009
C S J M University 								                                     Kanpur, India
Prediction of the binding site in HIV enzyme Reverse transcriptase, Protease, and Integrase.

BIOINFORMATICS & COMPUTATIONAL SKILLS
 
â€¢	Operating Systems & Platforms: Proficient in Windows and UNIX/Linux environments; experienced with command-line interfaces, high-performance cluster computing, cloud platforms (AWS), and containerization tools such as Docker, Conda, and Galaxy.
â€¢	Programming & Databases: Skilled in Python, R, and Perl programming languages; experience with MySQL and other database management systems.
â€¢	Next-Generation Sequencing (NGS) Analysis:
o	Single-cell RNA-Seq, whole transcriptome, whole exome, and methylome data analysis
o	Microbiome/Metagenomics: Proficient in QIIME2, Kraken2, MetaPhlAn4, HUMAnN3 for 16S and shotgun metagenomic data
o	Microarray and gene expression data processing and interpretation
â€¢	Molecular Data Analysis:
o	Sanger sequencing, fragment analysis, and MLPA interpretation using Coffalyser
â€¢	Functional Annotation & Pathway Analysis:
o	Tools and databases: GSEA, KEGG, STRING, DAVID, and other online bioinformatics resources
â€¢	Molecular Docking Tools:
o	Experienced with AutoDock, Molgro Virtual Docker, and Discovery Studio
â€¢	Code Repositories: 
o	GitHub: deppworld; Bitbucket: deppworld
PUBLICATION
 
â€¢	Verma, D., Siddharth, S., Yende, A. S., Wu, Q., & Sharma, D. LUCAT1-Mediated Competing Endogenous RNA (ceRNA) Network in Triple-Negative Breast Cancer. Cells, 2024; 13(22), 1918. https://doi.org/10.3390/cells13221918
â€¢	Verma D, Kapoor S, Kumari S, Sharma D, Singh J, Benjamin M, Bakhshi S, Seth R, Nayak B, Sharma A, Pramanik R, Palanichamy JK, Sivasubbu S, Scaria V, Arora M, Kumar R, Chopra A. Decoding the genetic symphony: Profiling protein-coding and long noncoding RNA expression in T-acute lymphoblastic leukemia for clinical insights. PNAS Nexus. 2024 Jan 12;3(2): pgae011. doi: 10.1093/pnasnexus/pgae011. PMID: 38328782; PMCID: PMC10847906. 
â€¢	Verma D, Kumar R, Ali MS, Singh J, Arora M, Singh I, Kumari S, Bakhshi S, Sharma A, Palanichamy JK, Tanwar P, Singh AR, Chopra A. BAALC gene expression tells a serious patient outcome tale in NPM1-wild type/FLT3-ITD negative cytogenetically normal-acute myeloid leukemia in adults. Blood Cells Mol Dis. 2022 Jul;95:102662. DOI: 10.1016/j.bcmd.2022.102662. Epub 2022 Apr 8. PMID: 35429905.
â€¢	Wu Q, Siddharth S, Verma D, Parida S, Sharma D; TRIM29 upregulation contributes to chemoresistance in triple negative breast cancer via modulating S100P-Î²catenin axis;Cell Communication and Signaling (in Revision).
â€¢	Nandi, D., Parida, S., Verma, D., Foley, J., Stewart, T., Korangath, P., Thatikonda, S., Wu, Q., Yi, M., Bishai, W. R., Ivkov, R., Sears, C. L., Casero, R. A., & Sharma, D. Spermine Oxidase as a Key Functional Node in Microbial Dysbiosis-Induced Breast Carcinogenesis. (In submission: Cell Host & Microbe)
â€¢	Ningombam A, Verma D, Kumar R, Singh J, Ali MS, Pandey AK, Singh I, Bakhshi S, Sharma A, Pushpam D, Palanichamy JK, Tanwar P, Ranjan Singh A, Chopra A. Refinement of Risk-Stratification of Cytogenetically Normal Acute Myeloid Leukemia Adult Patients by MN1 Expression. Asian Pac J Cancer Prev. 2024 Jul 1;25(7):2283-2289. doi: 10.31557/APJCP.2024.25.7.2283. PMID: 39068559; PMCID: PMC11480590. 
â€¢	Ningombam A, Verma D, Kumar R, Singh J, Ali MS, Pandey AK, Singh I, Bakhshi S, Sharma A, Pushpam D, Palanichamy JK, Tanwar P, Singh AR, Chopra A. Prognostic relevance of NPM1, CEBPA, and FLT3 mutations in cytogenetically normal adult AML patients. Am J Blood Res. 2023 Feb 15;13(1):28-43. PMID: 36937459.
â€¢	Arora M, Kumari S, Kadian L, Anupa G, Singh J, Kumar A, Verma D, Pramanik R, Kumar S, Yadav R, Chopra A, Chauhan SS. Involvement of DPP3 in modulating oncological features and oxidative stress response in esophageal squamous cell carcinoma. Biosci Rep. 2023 Sep 27;43(9):BSR20222472. doi: 10.1042/BSR20222472. PMID: 37531267; PMCID: PMC10500228.
â€¢	Kumari S, Ali MS, Singh J, Arora M, Verma D, Pandey AK, Benjamin M, Bakhshi S, Palanichamy JK, Sharma A, Singh I, Tanwar P, Singh AR, Pushpam D, Qamar I, Chopra A. Prognostic utility of key copy number alterations in T cell acute lymphoblastic leukemia. Hematol Oncol. 2022 Oct;40(4):577-587. doi: 10.1002/hon.3030. Epub 2022 Jun 4. PMID: 35644022.
â€¢	Singh J, Arora M, Kumari S, Verma D, Palanichamy JK, Qamar I, Chauhan SS, Chopra A, Molecular associations and clinical significance of core NHEJ pathway genes in renal clear cell carcinoma, Gene Reports, Volume 23, 2021, 101167, ISSN 2452-0144, https://doi.org/10.1016/j.genrep.2021.101167.
â€¢	Balasubramanian P, Singh J, Verma D, Kumar R, Bakhshi S, Tanwar P, Singh AR, Chopra A. Prognostic significance of CD45 antigen expression in pediatric acute lymphoblastic leukemia. Blood Cells Mol Dis. 2021 Jul;89:102562. doi: 10.1016/j.bcmd.2021.102562. Epub 2021 Mar 16. PMID: 33756412.
â€¢	Singh J, Kumari S, Arora M, Verma D, Palanichamy JK, Kumar R, Sharma G, Bakhshi S, Pushpam D, Ali MS, Ranjan A, Tanwar P, Chauhan SS, Singh A, Chopra A. Prognostic Relevance of Expression of EMP1,CASP1, and NLRP3 Genes in Pediatric B-Lineage Acute Lymphoblastic Leukemia. Front Oncol. 2021 Mar 5;11:606370. doi: 10.3389/fonc.2021.606370. PMID: 33747919; PMCID: PMC7973229.
â€¢	Singh J, Kumar R, Verma D, Rajput N, Palanichamy JK, Sharma G, Bakhshi S, Sharma A, Pushpam D, Seth R, Ranjan A, Tanwar P, Singh A, Arora M, Kumari S, Chopra A. MEF2C expression, but not absence of bi-allelic deletion of TCR gamma chains (ABD), is a predictor of patient outcome in Indian T-acute lymphoblastic leukemia. Am J Blood Res. 2020 Oct 15;10(5):294-304. PMID: 33224573; PMCID: PMC7675123.
â€¢	Balasubramanian P, Chopra A, Verma D, Singh IK, Kumar A, Sharma A, Kumar R. Imatinib resistance in chronic myeloid leukemia due to a rare mutation. Leuk Lymphoma. 2017 Jul;58(7):1750-1752. doi: 10.1080/10428194.2016.1256479. Epub 2016 Nov 20. PMID: 27868464.
â€¢	Misra A, Verma D, Chandramohan J, Bakhshi S, Kumar R, Gajendra S, Chopra A. Nuclear cupping in the blasts-more to the cup than myeloid. Hematol Oncol. 2016 Sep;34(3):171-3. doi: 10.1002/hon.2180. Epub 2014 Nov 5. PMID: 25370525.
â€¢	Chopra A, Soni S, Pati H, Kumar D, Diwedi R, Verma D, Vishwakama G, Bakhshi S, Kumar S, Gogia A, Kumar R. Nucleophosmin mutation analysis in acute myeloid leukaemia: Immunohistochemistry as a surrogate for molecular techniques. Indian J Med Res. 2016 Jun;143(6):763-768. doi: 10.4103/0971-5916.192027. PMID: 27748301; PMCID: PMC5094116.
â€¢	Chopra A, Soni S, Verma D, Kumar D, Dwivedi R, Vishwanathan A, Vishwakama G, Bakhshi S, Seth R, Gogia A, Kumar L, Kumar R. Prevalence of common fusion transcripts in acute lymphoblastic leukemia: A report of 304 cases. Asia Pac J Clin Oncol. 2015 Dec;11(4):293-8. doi: 10.1111/ajco.12400. Epub 2015 Aug 12. PMID: 26264145.
â€¢	Soni S, Chopra A, Bakhshi S, Vishwanath A, Verma D, Rai S, Kumar R. Prognostic impact of CD56 in pediatric AML. Int J Lab Hematol. 2015 Dec;37(6):e157-9. doi: 10.1111/ijlh.12402. Epub 2015 Jul 6. PMID: 26147745.

BOOK CHAPTER
 
Verma D and Kapoor S. Integrative Omics: Concept, Methodology, and Application. Chapter 11: Omics Approach for Personalized and Diagnostics Medicine. Elseviers Publications, 2024, Pages 175-185, ISBN 9780443160929, https:// doi.org/10.1016/B978-0-443-16092-9.00011-4.

GRANTS, AWARDS AND FELLOWSHIP
 
â€¢	ICMR-SRF-Research Associate Fellowship (2019-2021; INR 13,00,000)
â€¢	Scholarship for Highlights of Past EHA (HOPE) Asia 2020
â€¢	SGRF â€“ 2018 NGBT Gyan Scholarship Award at NGBT 2018, Jaipur India 
â€¢	SERB-DST International Travel Grant September 2018 
â€¢	IGGEB grant for ICGEB Course: Cancer Bioinformatics, Ismailia, Egypt.
TEACHING AND MENTORSHIP
 
â€¢	Mentored five graduate students during PhD, All India Institute of Medical Sciences, New Delhi, India 
â€¢	Mentored six masterâ€™s students during PhD, All India Institute of Medical Sciences, New Delhi, India 
â€¢	Mentored two masterâ€™s students during masterâ€™s degree, Department of Bioinformatics, UIET, CSJM University Kanpur, India 

CONFERENCE WORKSHOP TRAINING 
 
â€¢	EMBL Conference: From Functional Genomics to System Biology, EMBL Heidelberg, Germany.
â€¢	EMBL Course: Cancer Bioinformatics, EMBL Hinxton, United Kingdom
â€¢	ICGEB Course: Cancer Bioinformatics, Ismailia, Egypt. 
â€¢	Cancer Informatics Workshop, 28-30 Jan 2013, ACTREC Navi Mumbai, Supported by Department of Biotechnology, Govt. of India.
â€¢	Short term course on Bioinformatics, 22-23 Oct 2012, IIT Delhi, Supported by Department of Biotechnology, Govt. of India.
â€¢	Hands-On workshop on NGS, KMC Manipal University Manipal, India.
â€¢	NGS Workshop, MPAI 2016 PGI Chandigarh, India. 

PROFESSIONAL AFFILIATIONS 
 
â€¢	Executive Member:  Society of young Scientists, India (SYS)
â€¢	Associate Member: American Association for Cancer Research (AACR)
â€¢	Member: American Society of Gene & Cell Therapy (ASGCT)
â€¢	Ambassador and Early career: European Association for Cancer Research (EACR)
â€¢	Member European Hematology Association (EHA)
â€¢	Member: American Society of Clinical Oncology (ASCO)
â€¢	Member: International Society for Advancement of Cytometry (ISAC)

 

Sincerely,
Deepak Verma, PhD


